src/datajudge/utils.py

```diff
@@ -42,3 +42,143 @@ def format_difference(
         f"{s1[:diff_idx]}{_fmt_diff_part(s1, diff_idx)}",
         f"{s2[:diff_idx]}{_fmt_diff_part(s2, diff_idx)}",
     )
+
+
+class OutputProcessor(Protocol):
+    def __call__(  # noqa: E704
+        self,
+        collection: Collection,
+        counts: Optional[Collection] = None,
+    ) -> Collection: ...
+
+
+def output_processor_sort(
```

Comment:
Might it be useful to have unit tests for `output_processor_sort`, `output_processor_limit` and `sort_tuple_none_aware` in `datajudge/tests/unit/test_unit.py`?

Reply:
Done